packages/test/README.md

```diff
@@ -0,0 +1,30 @@
+# @theforeman/test
+
+> Adds theforeman testing tools to you project.
+
+[![Package Version](https://img.shields.io/npm/v/@theforeman/env.svg?style=flat-square)](https://www.npmjs.com/package/@theforeman/test)
+[![Downloads Status](https://img.shields.io/npm/dm/@theforeman/env.svg?style=flat-square)](https://npm-stat.com/charts.html?package=@theforeman/test&from=2016-04-01)
+[![Build Status: Linux](https://img.shields.io/travis/theforeman/foreman-js/master.svg?style=flat-square)](https://travis-ci.org/theforeman/foreman-js)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+
+
+## Installation
```

Comment:
Rather than installation, I expected to have more information about:
- Usage
  - How to write a simple test?
  - What tools can I import from `@theforeman/test` and how to use them? (e.g. shallow, mount, toJson)
  - How to run tests in the cloud? (Travis/Github Actions)